lib/hako/scripts/create_aws_cloud_watch_logs_log_group.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+require 'aws-sdk'
+require 'hako'
+require 'hako/script'
+
+module Hako
+  module Scripts
+    class CreateAwsCloudWatchLogsLogGroup < Script
+      # @param [Hash<String, Container>] containers
+      # @return [nil]
+      def deploy_starting(containers)
+        containers.each_value do |container|
+          log_configuration = container.log_configuration
+          unless log_configuration
+            next
+          end
+
+          if log_configuration[:log_driver] == 'awslogs'
+            create_log_group_if_not_exist(log_configuration.fetch(:options))
+          end
+        end
+      end
```

Comment:
Oh. I forgot about that.
The first was thinking, but had been forgotten.
I will add it too.